docs/DEV.md

```diff
@@ -258,10 +264,3 @@ _keyboard shortcuts_, etc.
 _**Visual Studio Code**_ (_**VSCode**_) is currently the most-popular code editor for reasons like
 being _lightweight_, _cleaner_, large marketplace of _extensions_, integrated _source control_
 features, _debugger_, _remote explorer_ support, _regular expression_ based find/replace, etc.
-
```

Comment:
Any reason you deleted this section?

Reply:
@walterbender, I accidentally removed it, i have now reverted the change.